src/elements/content-sidebar/activity-feed/app-activity/AppActivity.js

```diff
@@ -125,7 +125,7 @@ class AppActivity extends React.PureComponent<Props, State> {
                     'bcs-is-pending': isPending || error,
                 })}
                 data-resin-target="loaded"
-                data-resin-feature={`app_activity_${templateId}_card`}
+                data-resin-feature={`appActivityCard${templateId}`}
```

Comment:
Do we still want a delimiter between `appActivityCard` and the value of `templateId`?

Reply:
Ideally yeah, but we were running into issues ingesting the resin data when there was a delimiter. I moved `templateId` to the end of the string to make it easier to parse without delimiters